2019-06-23T19:32:56.1170666+01:00;VSTestExecutionThread;Info;SpecFlow+Runner execution started
2019-06-23T19:32:56.1170666+01:00;VSTestExecutionThread;Info;SpecFlow+Runner 3.0.352 in Framework clr40 in x64 mode execution started
2019-06-23T19:32:56.1170666+01:00;VSTestExecutionThread;Info;TestAdapter Location: C:\Repos\TandemTest\IssuesTestSolution\IssuesTestSolution\bin\Debug\TechTalk.SpecRun.VisualStudio.TestAdapter.dll
2019-06-23T19:32:56.1619436+01:00;VSTestExecutionThread;Info;SpecRun: running tests in C:\Repos\TandemTest\IssuesTestSolution\IssuesTestSolution\bin\Debug\IssuesTestSolution.dll
2019-06-23T19:32:56.1619436+01:00;VSTestExecutionThread;Info;Output folder configured to "C:\Repos\TandemTest\IssuesTestSolution\TestResults" (default)
2019-06-23T19:32:56.1619436+01:00;VSTestExecutionThread;Info;Profile: 
2019-06-23T19:32:56.2038324+01:00;VSTestExecutionThread;Info;Log file path: C:\Repos\TandemTest\IssuesTestSolution\TestResults\Unnamed project__2019-06-23T193256.log
2019-06-23T19:32:56.2058268+01:00;VSTestExecutionThread;Error;Invalid project license: The test assembly name must contain the licensed project name
2019-06-23T19:32:56.2187940+01:00;VSTestExecutionThread;Info;SpecFlow+Runner Evaluation Mode: Please purchase at http://www.specflow.org/plus to remove test execution delay.
2019-06-23T19:32:56.2258114+01:00;Thread#13;Info;Checking license
2019-06-23T19:32:56.2267704+01:00;Thread#13;Error;Invalid project license: The test assembly name must contain the licensed project name
2019-06-23T19:32:56.2277710+01:00;Thread#13;Error;Invalid project license: The test assembly name must contain the licensed project name
2019-06-23T19:32:56.2536991+01:00;Thread#13;Info;Evaluation mode: test execution starts in 6 seconds... (this delay is removed in the registered version)
2019-06-23T19:33:02.2550905+01:00;Thread#13;Info;Evaluation delay finished.
2019-06-23T19:33:02.2658648+01:00;Thread#13;Info;License check completed
2019-06-23T19:33:02.2658648+01:00;Thread#13;Info;starting test run
2019-06-23T19:33:02.2749011+01:00;Thread#13;Info;Discovering target: Default
2019-06-23T19:33:02.2758948+01:00;Thread#13;Info;Discovering tests from assembly: C:\Repos\TandemTest\IssuesTestSolution\IssuesTestSolution\bin\Debug\IssuesTestSolution.dll
2019-06-23T19:33:02.8897043+01:00:Appdomain Disposed
2019-06-23T19:33:02.8897043+01:00;Thread#13;Info;Discovered tests from assembly: C:\Repos\TandemTest\IssuesTestSolution\IssuesTestSolution\bin\Debug\IssuesTestSolution.dll
2019-06-23T19:33:02.8917017+01:00:Test Suite Dump
2019-06-23T19:33:02.8946926+01:00:#0: Unnamed project/ - 
2019-06-23T19:33:02.8946926+01:00:#1: Default - 
2019-06-23T19:33:02.8946926+01:00:#2: IssuesTestSolution - 
2019-06-23T19:33:02.8946926+01:00:#3: IssueTests - 
2019-06-23T19:33:02.9435610+01:00:#4: An authenticated client can create an issue - TestAssembly:IssuesTestSolution/Feature:IssueTests/Scenario:An+authenticated+client+can+create+an+issue
2019-06-23T19:33:02.9435610+01:00:#5: An authenticated client can edit an issue - TestAssembly:IssuesTestSolution/Feature:IssueTests/Scenario:An+authenticated+client+can+edit+an+issue
2019-06-23T19:33:02.9435610+01:00:#6: An authenticated client can list issues for a repo - TestAssembly:IssuesTestSolution/Feature:IssueTests/Scenario:An+authenticated+client+can+list+issues+for+a+repo
2019-06-23T19:33:02.9435610+01:00:End: Test Suite Dump
2019-06-23T19:33:02.9804620+01:00:Thread#0:Scheduler: get next test for thread 0
2019-06-23T19:33:02.9834535+01:00:Thread#:Scheduler: testitem #4 from normal test queue 
2019-06-23T19:33:02.9834535+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-06-23T19:33:02.9874819+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-06-23T19:33:02.9874819+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-06-23T19:33:02.9874819+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-06-23T19:33:02.9874819+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-06-23T19:33:02.9894397+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-06-23T19:33:02.9904709+01:00:Thread#0:Scheduler: with testItemExecutionIndex 0
2019-06-23T19:33:02.9934334+01:00:Thread #0: creating new test thread executor for IssuesTestSolution, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null (target: Default)
2019-06-23T19:33:02.9984129+01:00:Thread #0: Creating test assembly executor for execution model AppDomain
2019-06-23T19:33:02.9984129+01:00:Thread #0: Creating test assembly executor for execution model AppDomain
2019-06-23T19:33:02.9994097+01:00:using TechTalk.SpecRun.Framework.Execution.InProcess.InProcAppDomainTestAssemblyExecutorHost to execute tests
2019-06-23T19:33:03.0752080+01:00:Thread#0: Test #4/0 is started on thread #0
2019-06-23T19:33:03.5853383+01:00:Thread#0:T:-> Loading plugin C:\Repos\TandemTest\IssuesTestSolution\IssuesTestSolution\bin\Debug\SpecRun.Runtime.SpecFlowPlugin.dll
2019-06-23T19:33:03.5923194+01:00:Thread#0:T:-> Using app.config
2019-06-23T19:33:03.8191416+01:00:Thread#0:B:Given authenticated client is set for a create issue request
2019-06-23T19:33:04.0340826+01:00:Thread#0:T:done: IssueSteps.GivenAuthenticatedClientIsSetForACreateIssueRequest() (0.2s)
2019-06-23T19:33:04.0430970+01:00:Thread#0:B:When I send a post request with the following request body
2019-06-23T19:33:04.0430970+01:00:Thread#0:B:  --- table step argument ---
2019-06-23T19:33:04.0430970+01:00:Thread#0:B:  | name   | value                              |
2019-06-23T19:33:04.0440568+01:00:Thread#0:B:  | title  | Hi                                 |
2019-06-23T19:33:04.0440568+01:00:Thread#0:B:  | body   | Thus issue was menerated by a test |
2019-06-23T19:33:04.0440568+01:00:Thread#0:B:  | labels | bug                                |
2019-06-23T19:33:05.2303208+01:00:Thread#0:T:done: IssueSteps.WhenISendAPostRequestWithTheFollowingRequestBody(IssuesTestSolutio...) (1.1s)
2019-06-23T19:33:05.2313203+01:00:Thread#0:B:Then success response status should be 201
2019-06-23T19:33:05.4188164+01:00:Thread#0:T:error: Expected httpsRequests.GetResponseStatus(httpsRequests.restResponse) to be Created, but found Unauthorized.
2019-06-23T19:33:05.4198140+01:00:Thread#0:B:And response body
2019-06-23T19:33:05.4208115+01:00:Thread#0:T:skipped because of previous errors
2019-06-23T19:33:05.4347760+01:00:Thread#0:TestItem#4:Scheduler: reschedule tests for testitem #4
2019-06-23T19:33:05.4357728+01:00:Thread#0:Scheduler: get next test for thread 0
2019-06-23T19:33:05.4357728+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #4
2019-06-23T19:33:05.4357728+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-06-23T19:33:05.4357728+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-06-23T19:33:05.4357728+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-06-23T19:33:05.4357728+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-06-23T19:33:05.4357728+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-06-23T19:33:05.4357728+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-06-23T19:33:05.4357728+01:00:Thread#0:Scheduler: with testItemExecutionIndex 1
2019-06-23T19:33:05.4507718+01:00:Thread#0: Test #4/0 is finished on thread #0 as Failed within 2.3745655 seconds
2019-06-23T19:33:05.4537404+01:00:Thread#0: Test #4/1 is started on thread #0
2019-06-23T19:33:05.4537404+01:00:Thread#0:B:Given authenticated client is set for a create issue request
2019-06-23T19:33:05.4547221+01:00:Thread#0:T:done: IssueSteps.GivenAuthenticatedClientIsSetForACreateIssueRequest() (0.0s)
2019-06-23T19:33:05.4547221+01:00:Thread#0:B:When I send a post request with the following request body
2019-06-23T19:33:05.4547221+01:00:Thread#0:B:  --- table step argument ---
2019-06-23T19:33:05.4547221+01:00:Thread#0:B:  | name   | value                              |
2019-06-23T19:33:05.4547221+01:00:Thread#0:B:  | title  | Hi                                 |
2019-06-23T19:33:05.4557193+01:00:Thread#0:B:  | body   | Thus issue was menerated by a test |
2019-06-23T19:33:05.4557193+01:00:Thread#0:B:  | labels | bug                                |
2019-06-23T19:33:05.6058941+01:00:Thread#0:T:done: IssueSteps.WhenISendAPostRequestWithTheFollowingRequestBody(IssuesTestSolutio...) (0.1s)
2019-06-23T19:33:05.6066965+01:00:Thread#0:B:Then success response status should be 201
2019-06-23T19:33:05.6076948+01:00:Thread#0:T:error: Expected httpsRequests.GetResponseStatus(httpsRequests.restResponse) to be Created, but found Unauthorized.
2019-06-23T19:33:05.6076948+01:00:Thread#0:B:And response body
2019-06-23T19:33:05.6086916+01:00:Thread#0:T:skipped because of previous errors
2019-06-23T19:33:05.6086916+01:00:Thread#0:Scheduler: get next test for thread 0
2019-06-23T19:33:05.6086916+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #4
2019-06-23T19:33:05.6086916+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-06-23T19:33:05.6086916+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-06-23T19:33:05.6086916+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-06-23T19:33:05.6086916+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-06-23T19:33:05.6086916+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-06-23T19:33:05.6086916+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-06-23T19:33:05.6086916+01:00:Thread#0:Scheduler: with testItemExecutionIndex 2
2019-06-23T19:33:05.6096903+01:00:Thread#0: Test #4/1 is finished on thread #0 as Failed within 0.1559499 seconds
2019-06-23T19:33:05.6096903+01:00:Thread#0: Test #4/2 is started on thread #0
2019-06-23T19:33:05.6096903+01:00:Thread#0:B:Given authenticated client is set for a create issue request
2019-06-23T19:33:05.6106841+01:00:Thread#0:T:done: IssueSteps.GivenAuthenticatedClientIsSetForACreateIssueRequest() (0.0s)
2019-06-23T19:33:05.6116896+01:00:Thread#0:B:When I send a post request with the following request body
2019-06-23T19:33:05.6126989+01:00:Thread#0:B:  --- table step argument ---
2019-06-23T19:33:05.6126989+01:00:Thread#0:B:  | name   | value                              |
2019-06-23T19:33:05.6126989+01:00:Thread#0:B:  | title  | Hi                                 |
2019-06-23T19:33:05.6126989+01:00:Thread#0:B:  | body   | Thus issue was menerated by a test |
2019-06-23T19:33:05.6136761+01:00:Thread#0:B:  | labels | bug                                |
2019-06-23T19:33:05.7529797+01:00:Thread#0:T:done: IssueSteps.WhenISendAPostRequestWithTheFollowingRequestBody(IssuesTestSolutio...) (0.1s)
2019-06-23T19:33:05.7538081+01:00:Thread#0:B:Then success response status should be 201
2019-06-23T19:33:05.7548374+01:00:Thread#0:T:error: Expected httpsRequests.GetResponseStatus(httpsRequests.restResponse) to be Created, but found Unauthorized.
2019-06-23T19:33:05.7548374+01:00:Thread#0:B:And response body
2019-06-23T19:33:05.7548374+01:00:Thread#0:T:skipped because of previous errors
2019-06-23T19:33:05.7548374+01:00:Thread#0:Scheduler: get next test for thread 0
2019-06-23T19:33:05.7548374+01:00:Thread#:Scheduler: testitem #5 from normal test queue 
2019-06-23T19:33:05.7548374+01:00:Thread#0:Scheduler: considering testitem #5 for thread 0
2019-06-23T19:33:05.7548374+01:00:Thread#0:TestItem#5:testItemMatchesCurrentTestThread:UnSpecific
2019-06-23T19:33:05.7548374+01:00:Thread#0:TestItem#5:runsOnOneOtherThreadSpecific:False
2019-06-23T19:33:05.7548374+01:00:Thread#0:TestItem#5:runsOnOneOtherThreadUnspecific:False
2019-06-23T19:33:05.7548374+01:00:Thread#0:Scheduler: using testitem #5 for thread 0
2019-06-23T19:33:05.7548374+01:00:Thread#0:Scheduler: Test #5 is scheduled for thread
2019-06-23T19:33:05.7548374+01:00:Thread#0:Scheduler: with testItemExecutionIndex 0
2019-06-23T19:33:05.7558345+01:00:Thread#0: Test #4/2 is finished on thread #0 as Failed within 0.1461442 seconds
2019-06-23T19:33:05.7558345+01:00:Thread#0: Test #5/0 is started on thread #0
2019-06-23T19:33:05.7567940+01:00:Thread#0:B:Given authenticated client is set for an edit issue 1 request
2019-06-23T19:33:05.7577919+01:00:Thread#0:T:done: IssueSteps.GivenAuthenticatedClientIsSetForAnEditIssueRequest(1) (0.0s)
2019-06-23T19:33:05.7587895+01:00:Thread#0:B:When I send a patch request with the following request body
2019-06-23T19:33:05.7587895+01:00:Thread#0:B:  --- table step argument ---
2019-06-23T19:33:05.7587895+01:00:Thread#0:B:  | name   | value                              |
2019-06-23T19:33:05.7587895+01:00:Thread#0:B:  | title  | Hello                              |
2019-06-23T19:33:05.7587895+01:00:Thread#0:B:  | body   | This issue was generated by a test |
2019-06-23T19:33:05.7587895+01:00:Thread#0:B:  | labels | bug,Document                       |
2019-06-23T19:33:05.9039969+01:00:Thread#0:T:done: IssueSteps.WhenISendAPutRequestWithTheFollowingRequestBody(IssuesTestSolutio...) (0.1s)
2019-06-23T19:33:05.9039969+01:00:Thread#0:B:Then success response status should be 200
2019-06-23T19:33:05.9059951+01:00:Thread#0:T:error: Expected httpsRequests.GetResponseStatus(httpsRequests.restResponse) to be OK, but found Unauthorized.
2019-06-23T19:33:05.9059951+01:00:Thread#0:B:And response body
2019-06-23T19:33:05.9059951+01:00:Thread#0:T:skipped because of previous errors
2019-06-23T19:33:05.9059951+01:00:Thread#0:TestItem#5:Scheduler: reschedule tests for testitem #5
2019-06-23T19:33:05.9059951+01:00:Thread#0:Scheduler: get next test for thread 0
2019-06-23T19:33:05.9059951+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #5
2019-06-23T19:33:05.9059951+01:00:Thread#0:Scheduler: considering testitem #5 for thread 0
2019-06-23T19:33:05.9059951+01:00:Thread#0:TestItem#5:testItemMatchesCurrentTestThread:UnSpecific
2019-06-23T19:33:05.9059951+01:00:Thread#0:TestItem#5:runsOnOneOtherThreadSpecific:False
2019-06-23T19:33:05.9059951+01:00:Thread#0:TestItem#5:runsOnOneOtherThreadUnspecific:False
2019-06-23T19:33:05.9059951+01:00:Thread#0:Scheduler: using testitem #5 for thread 0
2019-06-23T19:33:05.9059951+01:00:Thread#0:Scheduler: Test #5 is scheduled for thread
2019-06-23T19:33:05.9059951+01:00:Thread#0:Scheduler: with testItemExecutionIndex 1
2019-06-23T19:33:05.9070527+01:00:Thread#0: Test #5/0 is finished on thread #0 as Failed within 0.1512182 seconds
2019-06-23T19:33:05.9070527+01:00:Thread#0: Test #5/1 is started on thread #0
2019-06-23T19:33:05.9070527+01:00:Thread#0:B:Given authenticated client is set for an edit issue 1 request
2019-06-23T19:33:05.9079906+01:00:Thread#0:T:done: IssueSteps.GivenAuthenticatedClientIsSetForAnEditIssueRequest(1) (0.0s)
2019-06-23T19:33:05.9079906+01:00:Thread#0:B:When I send a patch request with the following request body
2019-06-23T19:33:05.9079906+01:00:Thread#0:B:  --- table step argument ---
2019-06-23T19:33:05.9079906+01:00:Thread#0:B:  | name   | value                              |
2019-06-23T19:33:05.9079906+01:00:Thread#0:B:  | title  | Hello                              |
2019-06-23T19:33:05.9079906+01:00:Thread#0:B:  | body   | This issue was generated by a test |
2019-06-23T19:33:05.9079906+01:00:Thread#0:B:  | labels | bug,Document                       |
2019-06-23T19:33:06.0578154+01:00:Thread#0:T:done: IssueSteps.WhenISendAPutRequestWithTheFollowingRequestBody(IssuesTestSolutio...) (0.1s)
2019-06-23T19:33:06.0578154+01:00:Thread#0:B:Then success response status should be 200
2019-06-23T19:33:06.0598027+01:00:Thread#0:T:error: Expected httpsRequests.GetResponseStatus(httpsRequests.restResponse) to be OK, but found Unauthorized.
2019-06-23T19:33:06.0608017+01:00:Thread#0:B:And response body
2019-06-23T19:33:06.0618050+01:00:Thread#0:T:skipped because of previous errors
2019-06-23T19:33:06.0627954+01:00:Thread#0:Scheduler: get next test for thread 0
2019-06-23T19:33:06.0627954+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #5
2019-06-23T19:33:06.0627954+01:00:Thread#0:Scheduler: considering testitem #5 for thread 0
2019-06-23T19:33:06.0627954+01:00:Thread#0:TestItem#5:testItemMatchesCurrentTestThread:UnSpecific
2019-06-23T19:33:06.0627954+01:00:Thread#0:TestItem#5:runsOnOneOtherThreadSpecific:False
2019-06-23T19:33:06.0627954+01:00:Thread#0:TestItem#5:runsOnOneOtherThreadUnspecific:False
2019-06-23T19:33:06.0627954+01:00:Thread#0:Scheduler: using testitem #5 for thread 0
2019-06-23T19:33:06.0627954+01:00:Thread#0:Scheduler: Test #5 is scheduled for thread
2019-06-23T19:33:06.0627954+01:00:Thread#0:Scheduler: with testItemExecutionIndex 2
2019-06-23T19:33:06.0637929+01:00:Thread#0: Test #5/1 is finished on thread #0 as Failed within 0.1567402 seconds
2019-06-23T19:33:06.0637929+01:00:Thread#0: Test #5/2 is started on thread #0
2019-06-23T19:33:06.0637929+01:00:Thread#0:B:Given authenticated client is set for an edit issue 1 request
2019-06-23T19:33:06.0637929+01:00:Thread#0:T:done: IssueSteps.GivenAuthenticatedClientIsSetForAnEditIssueRequest(1) (0.0s)
2019-06-23T19:33:06.0647901+01:00:Thread#0:B:When I send a patch request with the following request body
2019-06-23T19:33:06.0647901+01:00:Thread#0:B:  --- table step argument ---
2019-06-23T19:33:06.0647901+01:00:Thread#0:B:  | name   | value                              |
2019-06-23T19:33:06.0647901+01:00:Thread#0:B:  | title  | Hello                              |
2019-06-23T19:33:06.0647901+01:00:Thread#0:B:  | body   | This issue was generated by a test |
2019-06-23T19:33:06.0647901+01:00:Thread#0:B:  | labels | bug,Document                       |
2019-06-23T19:33:06.2060439+01:00:Thread#0:T:done: IssueSteps.WhenISendAPutRequestWithTheFollowingRequestBody(IssuesTestSolutio...) (0.1s)
2019-06-23T19:33:06.2060439+01:00:Thread#0:B:Then success response status should be 200
2019-06-23T19:33:06.2080413+01:00:Thread#0:T:error: Expected httpsRequests.GetResponseStatus(httpsRequests.restResponse) to be OK, but found Unauthorized.
2019-06-23T19:33:06.2080413+01:00:Thread#0:B:And response body
2019-06-23T19:33:06.2080413+01:00:Thread#0:T:skipped because of previous errors
2019-06-23T19:33:06.2090381+01:00:Thread#0:Scheduler: get next test for thread 0
2019-06-23T19:33:06.2090381+01:00:Thread#:Scheduler: testitem #6 from normal test queue 
2019-06-23T19:33:06.2090381+01:00:Thread#0:Scheduler: considering testitem #6 for thread 0
2019-06-23T19:33:06.2090381+01:00:Thread#0:TestItem#6:testItemMatchesCurrentTestThread:UnSpecific
2019-06-23T19:33:06.2090381+01:00:Thread#0:TestItem#6:runsOnOneOtherThreadSpecific:False
2019-06-23T19:33:06.2090381+01:00:Thread#0:TestItem#6:runsOnOneOtherThreadUnspecific:False
2019-06-23T19:33:06.2090381+01:00:Thread#0:Scheduler: using testitem #6 for thread 0
2019-06-23T19:33:06.2090381+01:00:Thread#0:Scheduler: Test #6 is scheduled for thread
2019-06-23T19:33:06.2090381+01:00:Thread#0:Scheduler: with testItemExecutionIndex 0
2019-06-23T19:33:06.2090381+01:00:Thread#0: Test #5/2 is finished on thread #0 as Failed within 0.1452452 seconds
2019-06-23T19:33:06.2090381+01:00:Thread#0: Test #6/0 is started on thread #0
2019-06-23T19:33:06.2100353+01:00:Thread#0:B:Given authenticated client is set for a retrieve repo issues request
2019-06-23T19:33:06.2140237+01:00:Thread#0:T:done: IssueSteps.GivenAuthenticatedClientIsSetForARetrieveRepoIssuesRequest() (0.0s)
2019-06-23T19:33:06.2140237+01:00:Thread#0:B:When I send the get request
2019-06-23T19:33:06.3504682+01:00:Thread#0:T:done: IssueSteps.WhenISendTheGetRequest() (0.1s)
2019-06-23T19:33:06.3504682+01:00:Thread#0:B:Then success response status should be 200
2019-06-23T19:33:06.3524664+01:00:Thread#0:T:error: Expected httpsRequests.GetResponseStatus(httpsRequests.restResponse) to be OK, but found Unauthorized.
2019-06-23T19:33:06.3524664+01:00:Thread#0:B:And response body
2019-06-23T19:33:06.3524664+01:00:Thread#0:T:skipped because of previous errors
2019-06-23T19:33:06.3524664+01:00:Cancelling because configured number of failures reached
2019-06-23T19:33:06.3544604+01:00:Thread#0: Test #6/0 is finished on thread #0 as Failed within 0.1454223 seconds
2019-06-23T19:33:06.3694198+01:00:Appdomain Disposed
2019-06-23T19:33:06.3724128+01:00:Trying to cancel thread
2019-06-23T19:33:06.3724128+01:00:Thread cancelled
2019-06-23T19:33:06.3833857+01:00:Thread#-1:Scheduler: get next test for thread -1
2019-06-23T19:33:06.3833857+01:00:Thread#:Scheduler: at the end of normal test queue
2019-06-23T19:33:06.3833857+01:00:Thread#-1:Scheduler: enumerator for thread is at end
2019-06-23T19:33:06.3903657+01:00;Thread#13;Info;test run finished
2019-06-23T19:33:06.3903657+01:00;Thread#13;Info;publishing test results
2019-06-23T19:33:06.4172932+01:00;Thread#13;Info;test results published
2019-06-23T19:33:06.4172932+01:00;Thread#13;Info;generating reports
2019-06-23T19:33:06.5980967+01:00;Thread#13;Info;Starting external report generation process
2019-06-23T19:33:09.1051105+01:00;Thread#13;Info;Successfully generated reports.
2019-06-23T19:33:09.1061069+01:00;Thread#13;Info;Successfully generated reports
2019-06-23T19:33:09.1220657+01:00:Total: 3 (test executions: 7)
2019-06-23T19:33:09.1250558+01:00:Succeeded: 0
2019-06-23T19:33:09.1250558+01:00:Ignored: 0
2019-06-23T19:33:09.1290461+01:00:Pending: 0
2019-06-23T19:33:09.1290461+01:00:Skipped: 0
2019-06-23T19:33:09.1300444+01:00:Failed: 3
2019-06-23T19:33:09.1519849+01:00;VSTestExecutionThread;Info;Adding attachments to VSTest
2019-06-23T19:33:09.1519849+01:00;VSTestExecutionThread;Info;adding Test Execution Report 'C:\Repos\TandemTest\IssuesTestSolution\TestResults\Unnamed project__2019-06-23T193256.html'
========== End of log file ==========
